src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -88,6 +94,9 @@
     @Argument(fullName = StandardArgumentDefinitions.ADD_OUTPUT_SAM_PROGRAM_RECORD, shortName = StandardArgumentDefinitions.ADD_OUTPUT_SAM_PROGRAM_RECORD, doc = "If true, adds a PG tag to created SAM/BAM/CRAM files.", optional=true, common = true)
     public boolean addOutputSAMProgramRecord = true;
 
+    @Argument(fullName = StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, shortName = StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, doc = "If true, adds a command line header line to created VCF files.", optional=true, common = true)
+    public boolean addOutputVCFCommandLine = true;
```

Comment:
I still think it's weird to have boolean flags that default to true.  I would invert this so the command line input is `suppressOutputVCFCommandline` or something like that, but I feel like we've had this conversation before and the conclusion was that this is fine for some flags.